octopoes/bits/missing_headers/missing_headers.py

```diff
@@ -6,11 +6,26 @@
 from octopoes.models.ooi.web import HTTPHeader, HTTPResource
 
 
+def is_xss_capable(content_type: str) -> bool:
+    """Determine if the content type indicates XSS capability."""
+    xss_capable_types = [
+        "text/html",
+        "application/xhtml+xml",
+        "application/xml",
+        "text/xml",
+        "text/plain",
+        "image/svg+xml",
+    ]
```

Comment:
This can be moved outside the function as a `XSS_CAPABLE_TYPES` constant

Reply:
Even better if this could be configured bia a config ooi at some point.

Reply:
> Even better if this could be configured bia a config ooi at some point.

This is indeed an ideal candidate for using `Config`

Reply:
text/plain can be removed, as it cannot hold or reference executable content.

Reply:
I found this interesting discussion in which they highlight that browsers may misinterpret text/plain, rendering it, and enabling certain XSS attacks, I'm not sure If you agree though. It is only relevant for very old browsers imo.

Reply:
https://github.com/w3c/webappsec/issues/520

Reply:
True, its a possible side-effect of mimetype guessing. This is blocked by issuing the x-content-type-options header:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Content-Type-Options

Reply:
Okay! Ill remove the mime-type